examples/lobby_and_battle_servers.rs

```diff
@@ -146,31 +146,36 @@ impl<Config: ServerConfig> Default for ServerKeepAliveMap<Config> {
     }
 }
 
+#[derive(SystemSet, Clone, Hash, Debug, PartialEq, Eq)]
+enum SystemSets {
+    ServerRemoveTimedOutClients,
+}
+
 fn main() {
     App::new()
         .add_plugin(LogPlugin::default())
         .add_plugins(MinimalPlugins)
         // Lobby server
         .add_plugin(ServerPlugin::<LobbyConfig>::bind(LOBBY_SERVER))
-        .add_system(lobby_server_accept_new_connections.before("remove_timed_out"))
+        .add_system(
+            lobby_server_accept_new_connections.before(SystemSets::ServerRemoveTimedOutClients),
+        )
         .add_system(lobby_server_packet_handler)
         // Battle server
         .add_plugin(ServerPlugin::<BattleConfig>::bind(BATTLE_SERVER))
-        .add_system(battle_server_accept_new_connections.before("remove_timed_out"))
+        .add_system(
+            battle_server_accept_new_connections.before(SystemSets::ServerRemoveTimedOutClients),
+        )
         .add_system(battle_server_packet_handler)
```

Comment:
Please fix the formatting here, empty lines don't look good

Reply:
Cargo fmt, I think cause of long line.

Reply:
Oh, it was github that added an empty line in the diff view, sorry didn't notice it